workers/unity/Packages/io.improbable.gdk.core/Worker/ConnectionHandlers/MockConnectionHandler.cs

```diff
@@ -114,7 +114,6 @@ public MessagesToSend GetMessagesToSendContainer()
 
         public void PushMessagesToSend(MessagesToSend messages, NetFrameStats netFrameStats)
         {
-            throw new System.NotImplementedException();
```

Comment:
why did we remove this?

Reply:
The full stack runs (including the send system), which means this is being called. I'm not sure what we want to do with this right now, so just removing the error